pandas/core/config_init.py

```diff
@@ -460,6 +460,17 @@ def use_inf_as_na_cb(key):
 # Plotting
 # ---------
 
+plotting_backend_doc = """
+: str
+    The plotting backend to use. The default value is "matplotlib", the
+    backend provided with pandas. Other backends can be specified by
+    prodiving the name of the module that implements the backend.
+"""
+
+with cf.config_prefix('plotting'):
+    cf.register_option('backend', defval='matplotlib',
```

Comment:
see how excel is done
you can and should validate here

Reply:
makes sense, thanks for the feedback